sdk/cosmos/azure-cosmos-spark_3-0_2-12/pom.xml

```diff
@@ -279,6 +280,151 @@
           <skip>true</skip>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>3.2.4</version> <!-- {x-version-update;org.apache.maven.plugins:maven-shade-plugin;external_dependency} -->
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <relocations>
+                <relocation>
+                  <pattern>com.azure</pattern>
+                  <shadedPattern>${shadingPrefix}.com.azure</shadedPattern>
+                  <excludes>
+                    <exclude>com.azure.cosmos.spark.*</exclude>
+                    <exclude>com.azure.cosmos.implementation.SparkBridgeInternal</exclude>
+                    <exclude>com.azure.cosmos.models.CosmosParametrizedQuery</exclude>
```

Comment:
NIT: typo in the class name (unrelated, definitely not blocking)

Reply:
Thanks typo fixed.